src/main/java/org/folio/ld/dictionary/model/Resource.java

```diff
@@ -26,9 +25,7 @@ public class Resource {
 
   private JsonNode doc;
 
-  private UUID inventoryId;
-
-  private UUID srsId;
+  private InstanceMetadata instanceMetadata;
```

Comment:
Suggest to add
a getter with a null check to avoid null values of the Instance metadata object
"Unknown" enum to ResourceSource

Reply:
Hi @aleksei-pronichev,

Can you elaborate the comment? 
```
public InstanceMetadata getInstanceMetadata() {
  if (instanceMetadata == null) {
    // WHAT should be done here? Throw an exception? Or change the method signature to return an Optional?
  }
  return instanceMetadata;
}
```

Reply:
Also, I am not sure if we should add "UNKNOWN" source. We will have business logic around the source field (as below)

if (source == MARC) { do something }
else if (source == LINKED_DATA) { do somehthing else}

If someone set the source as UNKNOWN, then this logic will not work & hence the app will not work as expected (say, the instance won't get indexed)

Reply:
Agree, let's not add UNKNOWN source, it doesn't make sense

Reply:
discussed, resolved